backend/transcribee_backend/routers/document.py

```diff
@@ -253,10 +253,11 @@ def add_media_file(
         Task.document_id == document_id, Task.task_type == TaskType.REENCODE
     )
     task = session.exec(statement).one_or_none()
-    if task is None:
-        raise HTTPException(status_code=403)
-
-    if task.current_attempt.assigned_worker != authorized_worker:
+    if (
+        task is None
+        or task.current_attempt is None
+        or task.current_attempt.assigned_worker != authorized_worker
+    ):
         raise HTTPException(status_code=403)
```

Comment:
why do we answer with 'forbidden' here?

Reply:
What would you propose? 404?

Reply:
I decided for a 403 here, because this is a document endpoint and the document exists, so a 404 seems wrong. The problem is that the document may only be changed if a matching task exists that is assigned to this worker, which this checks for. If the task does not exist, the worker is not allowed to call this api, but another worker might. So I think 403 is the correct status code here

Reply:
It depends... If the user is not allowed to see the task a 404 should be the correct response IMO.